eng/pipelines/scripts/TypeSpec-Compare-CurrentToCodegeneration.ps1

```diff
@@ -0,0 +1,60 @@
+<#
```

Comment:
Lets not put scripts under the pipelines folder. Lets try to keep them consistently under `eng/scripts` which is where we have all the others.

Reply:
We might also want to consider this being put into the common set of scripts if other languages find it useful.

Reply:
Hi @weshaggard, thanks for your suggestion. I put it here because it does similar thing like this script: https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/scripts/Compare-CurrentToCodegeneration.ps1. One is for non-tsp generation, one is for tsp generation, so I put them together. Do you suggest moving both the two scripts to `eng/scripts` or only move the tsp one?

Reply:
Yes I think we should try to keep all the scripts under `eng/scripts`

Reply:
updated, putting two scripts to `eng/scripts` for now, we can put it to `common`  folder if later other languages found it useful.